Assets/AWSIM/Scripts/Environments/TrafficLight.cs

```diff
@@ -226,6 +226,8 @@ void Set(bool isLightOn)
         [SerializeField, Tooltip("Set the Renderer containing the bulb material.")] 
         new Renderer renderer;
 
+        public long LaneletElementID = 0;
```

Comment:
I think traffic light classes and lanelets should have no dependencies. How about attaching a component to the lanelet loader that represents the ID of the traffic light?